blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseBlobIntegrationTest.java

```diff
@@ -601,9 +603,16 @@ public void testPutInputStream() throws Exception {
 
    @Test(groups = { "integration", "live" })
    public void testPutMultipartByteSource() throws Exception {
-      long length = getMinimumMultipartBlobSize();
+      long length = 33554432 + 1; // MultipartUploadSlicingAlgorithm.DEFAULT_PART_SIZE + 1
+      BlobStore blobStore = view.getBlobStore();
+      MultipartUploadSlicingAlgorithm algorithm = new MultipartUploadSlicingAlgorithm(
+              getMinimumMultipartBlobSize(), blobStore.getMaximumMultipartPartSize(),
```

Comment:
Call `blobStore.getMinimumMultipartBlobSize` instead?  We should remove the test-specific variants now the `BlobStore` supports this.